core/endpoints.yml

```diff
@@ -1,5 +1,5 @@
 action_endpoint:
-  url: ${ACTION_SERVER_ENDPOINT}
+  url: http://action-server:8080/webhook
```

Comment:
Couldn't we start the action-server on port `80` instead?

Reply:
the docker container actually runs on non-root user, so we can't bind on 80.